src/cgame/cg_particles.cpp

```diff
@@ -664,6 +660,7 @@ static void CG_ParseValueAndVariance( const char *token, float *value, float *va
 
 		localValue = atof_neg( valueBuffer, allowNegative );
 
+		char *varEndPointer = nullptr;
 		varEndPointer = strchr( variancePtr, '%' );
```

Comment:
Could be one line instead of 2 I guess.

Reply:
Right, I've collapsed many of them but I missed this one! Done.